app/models/document.rb

```diff
@@ -0,0 +1,30 @@
+class Document < ApplicationRecord
+  has_many :content_items
+
+  validates :content_id, presence: true, uuid: true
+
+  validates :locale, inclusion: {
+    in: I18n.available_locales.map(&:to_s),
+    message: 'must be a supported locale'
+  }
+
+  def draft
```

Comment:
I think we can probably improve this somehow, maybe using a scope or similar. Do you think we should memoize here? I guess we hit it a lot, probably should act like a relation (I've been considering it one in my dozy just back from New Year state heh)